### PR TITLE
Add evicting disk KB (The CLI way)

### PR DESCRIPTION
**Pull request by bk201**
Related issue: https://github.com/harvester/harvester/issues/3364

Add a KB article for doing this with CLI.

The formal doc to remove a disk (GUI way) will be addressed in https://github.com/harvester/harvester/issues/3345.

### CODE REVIEW

**Review by Vicente-Cheng — APPROVED** [review]
just a nit, others LGTM. Thanks!

File: kb/2023-01-12/evict_replicas_from_a_disk.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+    image_url: https://github.com/bk201.png
+tags: [storage, longhorn, disk]
+hide_table_of_contents: false
+---
+
+Harvester replicates volumes data across disks in a cluster. Before removing a disk, the user needs to evict replicas on the disk to other disks to preserve the volumes' configured availability. For more information about eviction in Longhorn, please check [Evicting Replicas on Disabled Disks or Nodes](https://longhorn.io/docs/1.3.2/volumes-and-nodes/disks-or-nodes-eviction/).
+
+## Preparation
+
+The doc describes how to evict a Longhorn disk with the `kubectl` command. Before it, the user needs to make sure the environment is set up correctly.
```
Comment (guangbochen) [inline kb/2023-01-12/evict_replicas_from_a_disk.md:18]:
```suggestion
This document describes how to evict Longhorn disks using the `kubectl` command. Before that, users must ensure the environment is set up correctly.
```

File: kb/2023-01-12/evict_replicas_from_a_disk.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+There are two recommended ways to do this:
+
+1. Log in to any server role node and become root (`sudo -i`).
```
Comment (guangbochen) [inline kb/2023-01-12/evict_replicas_from_a_disk.md:21]:
```suggestion
1. Log in to any management node and switch to root (`sudo -i`).
```

File: kb/2023-01-12/evict_replicas_from_a_disk.md [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+    Run the command repeatedly, and the output should eventually become an empty map:
+    ```
+    {}
+    ```
+
+    This means Longhorn evicts replicas on the disk to other disks.
+
+    :::note
+    
+    If a replica always stays in a disk, please open the Longhorn GUI and check if there is free space on other disks.
```
Comment (guangbochen) [inline kb/2023-01-12/evict_replicas_from_a_disk.md:108]:
```suggestion
    If a replica always stays in a disk, please open the [Longhorn GUI](https://docs.harvesterhci.io/v1.1/troubleshooting/harvester#access-embedded-rancher-and-longhorn-dashboards) and check if there is free space on other disks.
```

**Review by guangbochen — APPROVED** [review]
LGTM, thanks.

**Pull request merged**